rust/rpmostree-client/src/lib.rs

```diff
@@ -53,7 +53,9 @@ pub struct Deployment {
     pub staged: Option<bool>,
     pub booted: bool,
     pub serial: u32,
-    pub origin: String,
+    pub ostree_refspec: Option<String>,
+    pub ostree_checksum: Option<String>,
+    pub container_image_reference: Option<String>,
```

Comment:
Hmm, looking at the CI failure, I wonder if there are other places (or users) that might rely on finding an `origin` key in the output of `rpm-ostree status --json` (and the corresponding `Deployments` D-Bus property of `org.projectatomic.rpmostree1.Sysroot`).

Reply:
Right, I think we need to weaken this to `pub origin: Option<String>` and then just add `pub container_image_reference: Option<String>`.  We can't rename the key because that would indeed potentially break existing clients.

This is related to https://github.com/coreos/rpm-ostree/issues/2839